backend/controllers/vehicleController.js

```diff
@@ -1,20 +1,31 @@
 const ErrorResponse = require('../utils/errorResponse');
 const asyncHandler = require('../middleware/async');
-const Vehicle = require('../models/Vehicle')
+const Vehicle = require('../models/Vehicle');
+const Dealership = require('../models/Dealership');
 
 // @desc        Get all vehicles for a specific dealership
 // @route       GET /api/v1/inventory/dealership/:dealershipId
 // @access      Private
 exports.getVehicles = asyncHandler(async (req, res, next) => {
+
+  // grab the dealership_ID from the body and verify that the dealership exists
+  const dealership = await Dealership.findById(req.params.dealershipId);
+
+  // no dealership found
+  if (!dealership) {
+      return next(
+          new ErrorResponse('dealership not found. Cannot return a list of vehicls without a valid dealership.', 404)
```

Comment:
404 error response message should have the ID of the entity you've failed to locate for debugging purposes.

Reply:
I have addressed this in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/45/commits/4218f055c5b933e587cbf4659e26f0c2d3ae6fb7)